Python/instrumentation.c

```diff
@@ -1,5 +1,6 @@
 #include "Python.h"
 
+#include "monitoring.h"
```

Comment:
Is it intentional that the new header file is not included in `Python.h`? This new API does not feel any special to me.